hack/jenkins/README.md

```diff
@@ -41,6 +41,39 @@ outside this repository, it's tricky to keep documentation for it up
 to date quickly. However, the scripts themselves attempt to provide
 color for the configuration(s) that each script runs in.
 
+## GCS Log Format
+
+Our `upload-to-gcs.sh` script runs at the start and end of every job. Logs on
+post-commit Jenkins go under `gs://kubernetes-jenkins/logs/`. Logs on PR
+Jenkins go under `gs://kubernetes-jenkins-pull/pr-logs/pull/PULL_NUMBER/`.
```

Comment:
are you talking about separate jenkins instances or separate triggers here?

might be worth a sketch of how these are different (eg: which jobs run post-commit vs per PR)